contracts/Auctions/DutchAuction.sol

```diff
@@ -295,6 +298,8 @@ contract DutchAuction is IMisoMarket, MISOAccessControls, BoringBatchable, SafeT
 
```

Comment:
`require(_beneficiary == msg.sender, "beneficiary must be sender")`

evtl? fands auch gruselig